src/app/[lang]/onboarding/introduce-yourself/introducePost.ts

```diff
@@ -0,0 +1,42 @@
+"use server";
+
+import { cookies } from "next/headers";
+import { redirect } from "next/navigation";
+import { env } from "@/env.mjs";
+
+type SignUpResponseError = {
+  fieldName: string;
+  fieldMessage: string;
+};
+
+export async function IntroducePost(introduce: string, lang: "en" | "uk") {
+  const jwtAccessTokenCookie = cookies().get("jwtAccessToken");
+
+  if (!jwtAccessTokenCookie) {
+    console.log("no access token");
+  }
+
+  const accessToken = jwtAccessTokenCookie!.value;
+
```

Comment:
Your code doesn't handle properly the absence of `jwtAccessToken`. You can at least redirect a user to a main page

```suggestion
  const jwtAccessTokenCookie = cookies().get("jwtAccessToken");

  if (!jwtAccessTokenCookie) {
    console.log("no access token");
    redirect(`/${lang}`);
  }
```